pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -1120,7 +1120,8 @@ var (
 	Secret = map[string]apiextensionsv1beta1.JSONSchemaProps{
 		"namespace": {
 			Description: "Namespace is the namespace in which the secret exists. If " +
-				"namespace is omitted, the namespace of the enclosing rule is assumed.",
+				"namespace is omitted, the namespace of the enclosing rule is assumed, " +
```

Comment:
Please bump `CustomResourceDefinitionSchemaVersion` in line 45

Reply:
Thanks, did not know about the schema version! Will bump!

Reply:
Done.